src/util/log.d

```diff
@@ -298,49 +304,47 @@ abstract class Logger
     }
 }
 
-class FileLogger(alias Layout) : Logger
+class FileLogger(Layout) : Logger
 {
-    // must be static to be thread-local
-    private static Appender!(char[]) buffer;
-
     private File file;
 
     // store cause it's awkward to derive in fileDescriptor, which must be signal-safe
     private int fileno;
 
-    this(string name, uint levels = LogLevel.info.orAbove)
+    private Layout layout;
+
+    mixin CachedAppender;
+
+    this(Layout layout, string name, uint levels = LogLevel.info.orAbove)
+    in (layout !is null)
     {
         super(levels);
-        file = File(name, "ab");
-        fileno = file.fileno;
+        this.file = File(name, "ab");
```

Comment:
Don't introduce company's coding standards:
it's still inconsistent and the diff is bigger than necessary.

Reply:
undone?